config/clients/go/template/README_calling_api.mustache

```diff
@@ -43,10 +43,12 @@ Get information about the current store.
 
 [API Documentation]({{apiDocsUrl}}/docs/api#/Stores/GetStore)
 
-> Requires a client initialized with a storeId
-
 ```golang
-store,  err := fgaClient.GetStore(context.Background()).Execute()
+options := ClientGetStoreOptions{
+    // You can rely on the store id set in the configuration or override it for this specific request
+    StoreId: "01FQH7V8BEG3GPQW93KTRFR8JB", 
```

Comment:
I think these should be something like: `{{packageName}}.PtrString("...")`,

Because they're pointers, not strings